client/src/pages/CheckoutPage/CheckoutPage.js

```diff
@@ -1,16 +1,55 @@
 import React from "react";
 import PropTypes from "prop-types";
 import Header from "../../components/Header";
+import { ContentWrapper } from "../../components/Wrapper";
+import CardSmall from "../../components/CardSmall";
+import CardMedium from "../../components/CardMedium";
+import Button from "../../components/Button";
 import NavbarBottom from "../../components/NavbarBottom";
+const axios = require("axios").default;
+
+const sendPostRequest = async (order) => {
+  try {
+    const res = await axios.post("/api/orders", order);
+    console.log(res.data);
+  } catch (err) {
+    // Error handling
+    console.error(err);
+  }
+};
 
 export const CheckoutPage = ({ title }) => {
+  const storedObjects = localStorage.getItem("blueCollarRocketCart");
```

Comment:
think you need a fallback
```suggestion
  const storedObjects = localStorage.getItem("blueCollarRocketCart") || "fallbackOption";
```